amethyst_assets/examples/hl.rs

```diff
@@ -177,6 +190,7 @@ fn main() {
         .add(RenderingSystem, "rendering", &[])
         .build();
 
-    let mut app = App::new(disp, "examples/assets/", State::Start);
+    let assets_dir = format!("{}/examples/assets/", env!("CARGO_MANIFEST_DIR"));
```

Comment:
I think this line will break on windows builds due to how it expects the path to be formatted. Maybe something like

```
let assets_dir = if cfg!(windows) {
    format!("{}\examples\assets\", env!("CARGO_MANIFEST_DIR"))
} else {
    format!("{}/examples/assets/", env!("CARGO_MANIFEST_DIR"))
};
```

Shamelessly copied from https://doc.rust-lang.org/std/macro.cfg.html

Reply:
I believe rust stod handles it.

Reply:
There won't be issues from forward slashes normally.